Features Branch: Finished the Mint Signup Page Clone

Ceejayski opened the pull request:
This project contains:

- A header with a formatted button at the top-right of the page body

- A centered form-wrapper class scaled to contain the form elements

- A form with inputs with appropriate types and labels

- A Footer

- The standard layout of a proper signup page

Ceejayski committed:
First commit

Added the index and css pages

Ceejayski committed:
widget-footer added

class widget footer was added to html page and styled

Ceejayski committed:
Footer tag and Linters Added

Added the footer of the page and stlyed it accordingly, also added Linters to the github-workflow

Ceejayski committed:
Added linters

Ceejayski committed:
Added openlock.png to the password input

Ceejayski committed:
Updates index.html

Did some changes on the widget-footer-span class

Ceejayski committed:
Added screenshot.JPG

Ceejayski committed:
Edited the README.md file and added the basic details

MuriloRoque commented on index.html:
```diff
@@ -0,0 +1,81 @@
+                        <input type="tel" name="phone" id="phone">
+                    </fieldset>
+                    <fieldset class="form-group">
+                        <label for="password">password</label>
```
This label should be capitalized ('Password' instead of 'password).

MuriloRoque reviewed (changes requested):
# Status: Changes requested 🔴 

## Hello @Ceejayski 👋 

## I am really impressed with your page, it's really awesome, almost identical to the original. But we still need to fix some details, alright? Please read carefully each one of my suggestions:

![here we go](https://thumbs.gfycat.com/FrankAdorableCollardlizard-size_restricted.gif)

# Page Structure

## Intuit logo and products

- [x] Please, add links to these texts and images, just like the original:

![Sem título4](https://user-images.githubusercontent.com/59691171/90663909-a5feee00-e220-11ea-942b-985d3b9fd726.png)

## Form

- [x] Please add this text below the Phone input, just like the original:

![Sem título](https://user-images.githubusercontent.com/59691171/90664002-caf36100-e220-11ea-906c-c69545d2e61d.png)

- [x] The password label should be capitalized, please check my comment in your code.

- [x] The 'Create Account' button has a padlock image, please put it in there:

![Sem título2](https://user-images.githubusercontent.com/59691171/90664146-f9713c00-e220-11ea-8250-ff811e801904.png)

## Bottom

- [x] Please increase the spacing between the lines in this text:

**Yours:**

![Yours](https://user-images.githubusercontent.com/59691171/90664277-245b9000-e221-11ea-830c-238877deeddf.png)

**Original:**

![Sem título5](https://user-images.githubusercontent.com/59691171/90664302-2cb3cb00-e221-11ea-9c0c-d611610afc82.png)

- [x] These texts below should be links, please fix it:

![Sem título3](https://user-images.githubusercontent.com/59691171/90664647-49e89980-e221-11ea-8943-452aacd1c378.png)

# General

- [x] You should add `fonts/fontawesome-free-5.14.0-web` folder to .gitignore since it's HUGE and you can simply reference a web link in your HTML < head > instead.

- [ ] You must fix your linter errors. If you ignore the folder I mentioned above, they will be almost all gone. But you should fix the linter errors in your `style.css` file anyway, alright? 😄 

# Summary

### You made a really nice job on this page, your README file, Pull Request and commits are professional, your CSS and HTML are using best practices, code linters are setup. You just need to fix those minor suggestions I made and you will be good to go 👍 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)

Ceejayski committed:
Made some corrections on the HTML and CSS pages

Adjusted the CSS indentation to the required standard on the linters styleint and also corrected some errors on the HTML page

Ceejayski committed:
Further corrections

Ceejayski committed:
Further Corrections on the CSS page

Ceejayski committed:
Further CSS correction

Ceejayski committed:
Solved Webint and Styleint Linters errors

Ceejayski committed:
Fixed Stlyelint errors

Ceejayski committed:
Fixed CSS errors

MuriloRoque reviewed (approved):
# Status: Approved 🟢 

## Hello again @Ceejayski 👋 

## Thank you for making all of the required changes I asked, you nailed it! Congratulations, you may merge, good luck with your next projects 🚀 

![congrats](https://media.giphy.com/media/8blb81XsdafLy/giphy.gif)

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)
